iis-wf/iis-wf-transformers/src/main/resources/eu/dnetlib/iis/wf/transformers/metadataextraction/checksum/postprocessing/meta/oozie_app/lib/scripts/transformer.pig

```diff
@@ -21,7 +21,7 @@ outputDocumentMeta = foreach joined generate
 		sourceDocumentMeta::year as year,
 		sourceDocumentMeta::publisher as publisher,
 		sourceDocumentMeta::references as references,
-		sourceDocumentMeta::authors as authors,
+		sourceDocumentMeta::authors as authorsMeta,
```

Comment:
Good to see that - it means problem was related to pig names only and we don't need to rename field in avro schema.

Does postprocessing integration test confirm `authors` are properly written? It seems `outputDocumentMeta` field name is `authorsMeta` from now on, is AvroStorage capable of matching it properly with `authors` field defined in avro schema?